lib/java/gradle.properties

```diff
@@ -25,14 +25,14 @@ maven-repository-url=https://repository.apache.org/service/local/staging/deploy/
 maven-repository-id=apache.releases.https
 
 # Dependency versions
-httpclient.version=4.5.13
-httpcore.version=4.4.15
+httpclient.version=5.2.1
+httpcore.version=5.2
```

Comment:
this is a breaking change, is this mandatory for the change?

Reply:
To migrate to Apache Http Components 5, it's mandatory but if it is only to migrate to Jakarta namespace, it's not.

I can create another pull request for the migration of Apache HttpComponent 4 to 5 but I think it's very important because :

- HttpClient5 is being actively maintained and HttpClient 4 is not (last important release was updated on Oct 06 2020, 4.5.14 was release in December 2022 but only with several minor correction on the core part) : [News Apache Http](https://hc.apache.org/news.html) so more and more developers are using 5.x version
- Framework like Spring Boot 3.x is using Apache HttpClient 5.x (for example in ClientHttpRequestFactory)

Reply:
Since these are discrete changes, I think it makes sense to keep them as separate PRs.

Reply:
@ctubbsii My goal was to be up to date for :

- Jakarta namespace
- HttpComponents 5 (actively maintained) and supported in latest version of framework like Spring Boot 3

But if it makes sense, I can split Jakarta and HttpComponents 5. @Jens-G @jeffalder what do you think ?

Reply:
@nicolasb29 I was just responding to your phrase:

> I can create another pull request for the migration of Apache HttpComponent 4 to 5

If they are not necessary to put in one PR, then putting in two would be easier to review.

Reply:
@ctubbsii - @Jimexist approved these changes 3 weeks ago, so splitting the PRs seems unwise. I'm just trying to understand what's next in the process, and how I can help make that happen.

Reply:
@jeffalder Okay, I took another look over the PR, and merged it.